proxy/src/discovery.rs

```diff
@@ -0,0 +1,34 @@
+//! Types and functionality related to Discovery
+
+use serde::Serialize;
+
+/// Representation of an untracked project
+#[derive(Serialize)]
+pub struct FeedItem {
```

Comment:
The fields in this struct are all project related, yet the name indicates a general purpose type. What is the to be expected needs for items in a feed?